pages/record/_.vue

```diff
@@ -66,13 +102,40 @@
     },
     data() {
       return {
+        altTitle: null,
+        description: null,
         error: null,
         image: null,
         fields: null,
-        media: null
+        media: null,
+        title: null,
+        edmIsShownBy: {}
```

Comment:
This needs to be re-removed. edm:isShownBy is included in the media array as the first item.